Casks/q/quba.rb

```diff
@@ -0,0 +1,21 @@
+cask "quba" do
+  version "1.4.0"
+  sha256 "4a30c194d59fc850650bd007a717745d53dd060e26792642f47e5368f373f1bb"
+
+  url "https://quba-viewer.org/files/Quba-#{version}-arm64.dmg"
+  name "quba"
```

Comment:
```suggestion
  name "Quba-Viewer"
```

Seems more consistent with the website.

Reply:
Fixed